motion_planning/include/motion_planning/planner_client.hpp

```diff
@@ -23,7 +23,7 @@ class planner_client_t
     const std::string service_name{ root + "/planner_service" };
     _service_client = nh.serviceClient<Service>(service_name);
     _obs_subscriber = nh.subscribe(root + "/pose", 1000, &planner_client_t::observation_callback, this);
-    _plan_publisher = nh.advertise<ml4kp_bridge::Plan>(root + "/ml4kp_plan", 1000, true);
+    _plan_publisher = nh.advertise<ml4kp_bridge::PlanStamped>(root + "/ml4kp_plan", 1000, true);
```

Comment:
Is there a reason to have `1000`? I have been using `1` (my understanding is that it has an impact on memory)